docker/demo/config/test-suite/delete-partition.yaml

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+dag_name: simple-deltastreamer.yaml
+dag_rounds: 1
+dag_intermittent_delay_mins: 1
+dag_content:
+  first_insert:
+    config:
+      record_size: 1000
+      num_partitions_insert: 5
+      repeat_count: 1
+      num_records_insert: 10
+    type: SparkInsertNode
+    deps: none
+  first_delete_partition:
+    config:
+      partitions_to_delete: "1969/12/31"
+    type: SparkDeletePartitionNode
+    deps: first_insert
+  second_validate:
+    config:
+      validate_full_data : true
+      partitions_to_skip_validate : "0"
```

Comment:
optional: should we rename this to `input_partitions_to_skip_validate`? It wasn't clear to me why this has '0' value when hudi table has date partitions until I read the code in BaseValidateDatasetNode.